src/awesome_map/BoundaryInterceptor.js

```diff
@@ -151,6 +163,28 @@ define(function(require) {
         this._easing = options.easing || EasingFunctions.easeOutQuart;
 
         /**
+         * This value defines the minimum number of pixels beyond a boundary a scroll event
+         * must reach for boundary events to be fired, so that light touches and small scrolls 
+         * (by default) wont fire events constantly.  A value of 0 will cause every event which
+         * passes a boundary to fire a boundary event.
+         * @type {number}
+         * @private
```

Comment:
Oh, geez, sorry.  I was thinking this would be up at, say, line 122.  And, since this is now documented, the constant probably does not need a comment.  That's what I get for commenting top to bottom like an English speaker.  :disappointed: